db/migrate/20220214173716_create_merchants.rb

```diff
@@ -0,0 +1,9 @@
+class CreateMerchants < ActiveRecord::Migration[5.2]
```

Comment:
Since we're getting our merchant data from an API, I don't believe we have a need to create a `merchants` table in our db. If we were to decide to save data that we get from the API, then we would want to create a table for this.